app/views/users/new.html.erb

```diff
@@ -0,0 +1 @@
+<p>Create a new user <button>New user</button></p>
```

Comment:
- [ ] This file is not needed.  We only need "index" and "show" files.  Kindly delete this file.